fix (a11y): add nav landmark to social and legal links

noa-santo opened the pull request:
Changes: 
- add nav landmark to the legal links in the menu
- add nav landmark to the social links in the menu

Commit:
fix (a11y): add nav tag to "Data Protection" & "Imprint" in menu

georgiee reviewed (changes requested):
Thanks for bringing this up! The idea is correct but I think there is an issue with the depth of the placement.

The `nav` tag is far too low by wrapping individual links. It should at least wrap the `LegalInks`. Maybe even higher to catch the social links.

Please gather some knowledge about navigation landmarks so you can decide properly. Also ensure each nav element has a unique aria-label or aria-labelledby attribute as we have more than one `nav` then.

Some starters I found:
+ https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/navigation.html
+ https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/

Commit:
fix (a11y): add aria-label to landmarks
fix (a11y): wrap whole legal links in nav landmark instead of wrapping the links individually
fix (a11y): wrap social links in nav landmark
